packages/react-i18n/README.md

```diff
@@ -0,0 +1,80 @@
+# `@shopify/react-i18n`
+
+[![Build Status](https://travis-ci.org/Shopify/quilt.svg?branch=master)](https://travis-ci.org/Shopify/quilt)
+[![License: MIT](https://img.shields.io/badge/License-MIT-green.svg)](LICENSE.md) [![npm version](https://badge.fury.io/js/%40shopify%2Freact-i18n.svg)](https://badge.fury.io/js/%40shopify%2Freact-i18n.svg) [![npm bundle size (minified + gzip)](https://img.shields.io/bundlephobia/minzip/@shopify/react-i18n.svg)](https://img.shields.io/bundlephobia/minzip/@shopify/react-i18n.svg)
+
+I18n library for React
```

Comment:
why should we use this package over https://github.com/i18next/react-i18next? What additional benefits does this provide? I'd use that as a basis for the description here.

Reply:
@lemonmade @TheMallen may be able to provide more context

Reply:
Picking a specific third party library to compare to in particular is a little weird, there are 100s of libraries for everything in the react ecosystem. I think it can be useful to do comparison / contrast to popular libraries.

We're going to want to revisit the readme when Chris returns regardless since he has the most context on it.

Reply:
There was some discussion at the beginning of the year around custom vs third-party libraries: https://github.com/Shopify/web-foundation/pull/3#issuecomment-364538838, but agree Chris is going to have a lot of input on this Readme either way.